core/dbt/common/events/__init__.py

```diff
@@ -0,0 +1,9 @@
+from dbt.common.events.base_types import EventLevel
+from dbt.common.events.event_manager_client import get_event_manager
+from dbt.common.events.functions import get_stdout_config
+from dbt.common.events.logger import LineFormat
+
+# make sure event manager starts with a logger
+get_event_manager().add_logger(
```

Comment:
this behavior was in `factory.py`, we have moved it to __init__ as this is a more idiomatic place to locate behavior that we expect to happen on module import